### DISCUSSION TITLE
Rate limit per day was exceeded and it is not renewed at all after few days

### DISCUSSION THREAD

**Discussion by SrdjanPaunovic** [etsy/open-api#680] [Q&A]
I facing some weird situations and I want to check with you guys about what's wrong.
The user exceeded his daily limit. We have some retry mechanisms that retry to repeat requests after 5 minutes. It past 2 days but this user still can't send requests. 

In the response header we always got this: 
```
X-Remaining-Today: 0
Retry-After: 86400
```

![image](https://user-images.githubusercontent.com/22815104/204238521-ae7cb889-dea0-4408-8517-10581fa8e7ec.png)

Does this 24-hour interval reset if we try to submit a request before 24 hours have passed?  I read [documentation](https://developers.etsy.com/documentation/essentials/rate-limits) and I couldn't find an explanation for this behavior.

Here is `X-Etsy-Request-Uuid: EupgzKdscLNcyTakcWdWQ63ygf4e` from the header.

**Comment by etsyachristensen** [comment]
@SrdjanPaunovic We apologize for the delay in getting back to you. Are you still having this issue? Is it just the one shop?
